gensim/models/word2vec.py

```diff
@@ -1180,9 +1180,16 @@ def intersect_word2vec_format(self, fname, lockf=0.0, binary=False, encoding='ut
                         self.wv.syn0[self.wv.vocab[word].index] = weights
         logger.info("merged %d vectors into %s matrix from %s" % (overlap_count, self.wv.syn0.shape, fname))
 
+    def get_words_from_vocab(self):
```

Comment:
hmm, I was confused at first myself but then I just did it to maintain coding style of word2vec.

Do you think it might be confusing for newcomers if we don't have the forwarding methods. It took me quite a while to figure KeyedVectors as having all after training functionality as word2vec!

Also, most of the word2vec functions don't have `docstring` and one has to read the `docstring` of w2v.wv to figure things out. That might throw people off. Would you like me to add docstrings for forwarding functions?